src/redist/targets/BundledTemplates.targets

```diff
@@ -86,10 +86,10 @@
 
   <ItemGroup>
     <BundledTemplates Include="@(Bundled60Templates)" TemplateFrameworkVersion="6.0"/>
-    <BundledTemplates Include="@(Bundled50Templates)" TemplateFrameworkVersion="5.0"/>
-    <BundledTemplates Include="@(Bundled31Templates)" TemplateFrameworkVersion="3.1"/>
-    <BundledTemplates Include="@(Bundled30Templates)" TemplateFrameworkVersion="3.0"/>
-    <BundledTemplates Include="@(Bundled21Templates)" TemplateFrameworkVersion="2.1"/>
+    <BundledTemplates Include="@(Bundled50Templates)" TemplateFrameworkVersion="5.0" Condition="$(ProductMonikerRid.StartsWith('win'))" />
```

Comment:
Would it make sense to pull this out to two ItemGroups, one with the `Condition="$(ProductMonikerRid.StartsWith('win'))"`, so we don't have to repeat it so much?